src/strategies/subgraph-split-delegation/index.ts

```diff
@@ -0,0 +1,132 @@
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest, getScoresDirect } from '../../utils';
+import { Strategy } from '@snapshot-labs/snapshot.js/dist/src/voting/types';
+
+export const author = 'aragon';
+export const version = '0.1.0';
+
+const DEFAULT_BACKEND_URL =
+  'https://api.studio.thegraph.com/query/87073/split-delegation/version/latest';
```

Comment:
Hi @xavikh could you share the subgraph id for this? these test URLs are rate limited

Reply:
We haven't published the subgraph yet. We were planing on passing the final URL through the strategy options.